src/farcasterd/trade_state_machine.rs

```diff
@@ -475,8 +494,8 @@ fn attempt_transition_to_taker_connect_or_take_offer(
     match event.request.clone() {
         BusMsg::Ctl(CtlMsg::TakeOffer(PubOffer {
             public_offer,
-            external_address,
-            internal_address,
+            bitcoin_address: external_address,
+            monero_address: internal_address,
```

Comment:
I'd get rid of renaming to external/internal address here entirely since
1. imo, those names were conceptually misleading to begin with
2. they're just being renamed to `arb_addr`/`acc_addr` thereafter anyway

Reply:
ffde2b9be5584b15d1ba44ffbaf8a7153cf8e1b0